cogs/generalcog.py

```diff
@@ -65,7 +65,24 @@ async def git(self, ctx, *args):
                     colour=0x1e2327)
         await ctx.send(embed=emb)
 
-
+    @commands.command()
+    async def pokies(self, ctx, *args):
+        # Get all custom Emoji from server
+        allEmotes = self.bot.emojis
```

Comment:
This will sample from all emojis that the bot has access to. This means you will get emojis from other servers as well. I'd prefer to sample from [`Guild.emojis`](https://discordpy.readthedocs.io/en/latest/api.html#discord.Guild.emojis) instead.

Also, prefer snake_case `all_emotes`

Reply:
Updated.